docs/en/Architecture.md

```diff
@@ -32,13 +32,13 @@ can serve as a unifying layer for any number of varied data sources.
 Alluxio's design uses a single primary master and multiple workers. At a very high level, Alluxio
 can be divided into three components, the [master](#master), [workers](#worker), and
 [clients](#client). The master and workers together make up the Alluxio servers, which are the
-components a system admin would maintain and manage. The clients are generally the
+components a system admin would maintain and manage. The Alluxio clients reside within the
 applications, such as Spark or MapReduce jobs, or Alluxio command-line users. Users of Alluxio
-will usually only need to interact with the client portion of Alluxio.
+will typically only need to interact with the client portion of Alluxio.
 
 ### Master
 
-Alluxio master can be deployed as primary master or secondary master.
+Alluxio masters can be deployed as a primary master or a secondary master.
```

Comment:
This isn't very accurate - typically users deploy a master without choosing primary vs secondary, and master election is done to decide which master will temporarily be primary. We should either fix the accuracy or remove the discussion of deployment and stick only with the architecture.

Reply:
I modified it a bit, and removed the deployment part.